keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -75,7 +75,7 @@ class MemoryCloudDriverCache(
 
       // TODO-AJ should be able to swap this out for a call to `/search`
       cloudDriver
-        .getSecurityGroupSummaries(account, credential.type, region)
+        .getSecurityGroupSummaries(account, credential.type, region, DEFAULT_SERVICE_ACCOUNT)
```

Comment:
I opted for specifying the default service account explicitly in every call (despite that being the default value for the argument), to make it clear where we're using the default vs. using the service account associated with the delivery config.